enterprise/internal/campaigns/store_changesets.go

```diff
@@ -397,11 +405,12 @@ type ListChangesetsOpts struct {
 	OnlyWithoutDiffStats bool
 	OnlySynced           bool
 	ExternalServiceID    string
+	WithoutHistory       bool
```

Comment:
Nitpick, but feel free to ignore, because we have `OnlyWithoutDiffStats` and `WithoutDeleted` above 😐

```suggestion
	OnlyWithoutHistory       bool
```